DockerHubBackend/DockerHubBackend/appsettings.json

```diff
@@ -8,5 +8,10 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "DefaultConnection": "Host=localhost;Port=5432;Database=uks-database;Username=admin;Password=admin"
+  },
+  "Jwt": {
+    "Key": "D47C7FD6E2D7DVM20VSX1YVP6289QBWN62MX33FIUH",
```

Comment:
Should we add this values to an env file?

Reply:
Yes, both jwt key and database connection string should be environment variables. I had a trouble making it work so I just left it like this for now.